src/main/java/trivia/core/question/QuestionService.java

```diff
@@ -0,0 +1,23 @@
+package trivia.core.question;
+
+import java.util.EnumMap;
+import java.util.Map;
+
+public class QuestionService {
```

Comment:
Opet kad pogledas u realnu igru, igra ima spilove karata. U terminologiji zadatka se koristi Question Deck da opise spilove karata, i to je dobar hint da treba modelovati takav objekat.

To bi onda bio pravi rich objekat, i iz samog imena QuestionsDecks bi se radjala opisnija imena metoda kao sto su "izvuci pitanje" ili "kreiraj deck sa ovoliko pitanja". Tako bi cak i taj broj pitanja (50) mogao da se prosledi iz Game. klase, i kada pocinjemo Game mozemo da odlucimo koliko pitanja da stavimo.